src/pages/docs/auth/token.mdx

```diff
@@ -23,6 +23,21 @@ One of the important benefits of using an Ably SDK is that the automatic refresh
 
 To use automatic refresh of tokens, provide either an `authUrl` or an `authCallback`. When the token is near to expiry the `authUrl` or `authCallback` is invoked and a new token is automatically requested.
 
+### Automatic renewal timing
```

Comment:
I don't think this is necessary as it's discussed here: https://ably.com/docs/auth/token#refresh

Reply:
See 4ba9bdfef6bf489ef2bdf24de2f95d61f424e85f